### ISSUE TITLE
Windows 10 No permissions EXE deletes itself. [SOLVED]

### ISSUE THREAD

**Issue by BiacTYK** [Amulet-Team/Amulet-Map-Editor#140]
**Description of the bug**
When attempting to run the amulet_app.exe I receive the notification stating:
"Windows cannot access the specified device, path, or file. You may not have the appropriate permissions to access the item."
The file then deletes itself from this plane of existence. It is not hidden in the folder, nor is it in my Recycle Bin.

**To Reproduce**
1. Attempt to run the program on my computer, I guess.

**Expected behavior**
The expected behaviour is an application launching.

**Screenshots**
![error](https://user-images.githubusercontent.com/51229104/100698427-047a5300-335e-11eb-81c0-7581c88e2904.png)

**Desktop**
 - OS: Windows 10 Pro V1909 Build 18363.1198
- Ryzen 7 3700x
- 32Gb DDr4 TridentZ
- M.2 NVME Samsung 970pro 256Gb
- WDBlue 2TB HDD
- RTX 2070Super
- Asus Prime X-570p

**Additional context**
This issue persists regardless of what drive I attempt to run it from, and regardless if it is in its archive or not. I have used CMD to take ownership of the Amulet folder with no change in behavior. I have also told Windows to run the .exe as administrator with no change in behavior. Once I run the program, all that happens is the error, screenshot attached, and the .exe disappears from my system.

**Comment by Podshot** [comment]
Hmm, that is really odd, especially that the file get removed. Do you have an anti-virus program running that is auto removing the file(s) when you attempt to run Amulet?

**Comment by BiacTYK** [comment]
> Hmm, that is really odd, especially that the file get removed. Do you have an anti-virus program running that is auto removing the file(s) when you attempt to run Amulet?

I do have Avast Anti-virus, but it is not set to run with windows. I have its service disabled. Windows defender may be taking it's place in the absence of Avast and could perhaps be the cause of my issue. I'll look into this here shortly.

**Comment by BiacTYK** [comment]
Ok.. This is exceptionally odd. I looked into my Windows defender settings and did not see any simple toggles I could have used to turn it off. Regrettably I then opened avast to check it's settings were the same as I had them. Self-defence off, auto-start off, services disabled on windows start-up, ect. For some odd reason Amulet now opens with no issue. I wish I had tried to run the program *before* opening Avast as I changed to many variables at once to know what may have caused the program to start working. In desperation to find the actual solution I rebooted my machine and went straight to launch Amulet; Again it launched no problem. This is quiet weird. I'm going to chock it up to "The electronic mind". I'm coining that to mean 'when a computer system does whatever it feels like doing for no discernible reason.' I googled and couldn't find a word for this behavior but I've experienced "the electronic mind" before..
